compiler/src/dotty/tools/dotc/core/classfile/ClassfileParser.scala

```diff
@@ -761,7 +761,7 @@ class ClassfileParser(
           assert(cond,
             s"Unpickling ${classRoot.symbol.showLocated} from ${classRoot.symbol.associatedFile} is not allowed with -Yscala2-unpickler $allowed")
 
-        if (allowed != "always") {
+        if (false && allowed != "always") {
```

Comment:
Should be removed

Reply:
Ah, I needed that because my local setup does not run otherwise. I have to inquire why.